osm2streets/src/road.rs

```diff
@@ -564,6 +564,9 @@ pub(crate) struct RoadEdge {
     /// Pointed into the intersection
     pub pl: PolyLine,
     pub lane: LaneSpec,
+    /// Which edge of a road? Note this is an abuse of DrivingSide; this just means the left or
```

Comment:
This was a bit of intermediate debugging that was particularly helpful, just carrying it along for next time